kubernetes/cray-ims/templates/cray-ims-v2-image-create-kiwi-ng-job-template.yaml

```diff
@@ -44,29 +44,7 @@ data:
         spec:
           restartPolicy: Never  # Don't ever restart this job
           initContainers:
-          # Step 1: Set ownership of volume mounts
```

Comment:
Even if ims-utils must run as root, does it hurt to leave these volume mounts owned by 'nobody'?

Reply:
Okay. I thought leaving it would make it easier if we found a way later to return to the 'nobody' user. Feel free to resolve this comment.